game-core/src/main/java/games/strategy/engine/lobby/client/login/LobbyServerProperties.java

```diff
@@ -71,9 +71,10 @@
   public URI getHttpsServerUri() {
     try {
       return new URIBuilder()
+          // TODO: use https
           .setScheme(
               // allow env variable override of https so we can do local development with http
-              Optional.ofNullable(System.getenv("HTTP_SERVER_PROTOCOL")).orElse("https"))
+              Optional.ofNullable(System.getenv("HTTP_SERVER_PROTOCOL")).orElse("http"))
```

Comment:
I'm not sure if having local development set a special environment variable is the best way for us to override to http. For now, the protocol is set to http to keep things simple and working.